### PR TITLE
op-node: Fillbytes size check in l1 block info

### PR DESCRIPTION
**Pull request by maurelian**
**Description**

Prevents unhandled panics in the case that unexpectedly large values are passed to `FillBytes`.

**Tests**

No new tests, it's not clear that these errors are actually reachable.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**
- Fixes ENG-2613

### CODE REVIEW

File: op-node/rollup/derive/deposit_log_test.go [hunk 1]
```diff
@@ -47,8 +50,10 @@ type receiptData struct {
 	DepositLogs []bool
 }
 
-func makeReceipts(rng *rand.Rand, blockHash common.Hash, depositContractAddr common.Address, testReceipts []receiptData) (receipts []*types.Receipt, expectedDeposits []*types.DepositTx) {
+func makeReceipts(rng *rand.Rand, blockHash common.Hash, depositContractAddr common.Address, testReceipts []receiptData) ([]*types.Receipt, []*types.DepositTx, error) {
 	logIndex := uint(0)
+	receipts := []*types.Receipt{}
+	expectedDeposits := []*types.DepositTx{}
```
Comment (maurelian) [inline op-node/rollup/derive/deposit_log_test.go:56]: I realize this is kind of ugly, but the alternative would've been to name the error variable in the function declaration, which seemed like it went against convention in the rest of the codebase.

Comment (trianglesphere) [reply op-node/rollup/derive/deposit_log_test.go:56]: we've done it, but I actually prefer explicit rather than named returns

**Review by trianglesphere — APPROVED** [review]
lgtm

File: op-node/rollup/derive/deposit_log_test.go (hunk 1 above)
Comment (maurelian) [reply op-node/rollup/derive/deposit_log_test.go:56]: Yeah, this instance of named returns seemed unconventional as well.

**Review by mslipper — APPROVED** [review]

**Pull request merged**